packages/exec/src/exec.ts

```diff
@@ -1,6 +1,8 @@
 import * as im from './interfaces'
 import * as tr from './toolrunner'
 
+export {ExecOptions} from './interfaces'
```

Comment:
should this import the interface, and then export? for example, see [here](https://github.com/actions/toolkit/blob/master/packages/glob/src/glob.ts)

that way the `exec` function can be declared as:

`export declare function exec(commandLine: string, args?: string[], options?: ExecOptions): Promise<number>;`

(better .d.ts)

Reply:
It looks better for me too. I'll update this branch.